app/experimenter/base/tests/test_serializers.py

```diff
@@ -16,3 +24,13 @@ def test_serializer_outputs_expected_schema(self):
         country = CountryFactory.create()
         serializer = CountrySerializer(country)
         self.assertEqual(serializer.data, {"code": country.code, "name": country.name})
+
+
+class TestLanguageSerializer(TestCase):
+    def test_serializer_outputs_expected_schema(self):
+        language = LanguageFactory.create()
+        serializer = LanguageSerializer(language)
+        self.assertEqual(
+            str(language), f"{serializer.data['name']} ({serializer.data['code']})"
```

Comment:
ahhh so this was the problem! why do we need `str(language)` but not `str(locale)`? Is there some change that we should be making to the factory to make this a simple `assert`?

Reply:
so the reason is why we need this in `language `and not in `locale or country `is because we have some old test cases which are testing that string. for instance : 
<img width="862" alt="image" src="https://user-images.githubusercontent.com/104033388/166507755-703fda48-bec8-4bd2-9f5d-63a02ecbbf5d.png">